<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgACDMember</class>
 <widget class="QDialog" name="DlgACDMember">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>358</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ACD/Ring group members</string>
  </property>
  <widget class="QListWidget" name="lstExtns">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>181</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lstMembers">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>181</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbACDName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>0</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>ACD:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblExtns">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>List of Extension not Assigned</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMembers">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>List of Member</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pbSave">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
